Application/EdFi.Ods.Api/Security/Authorization/AuthorizationBasisMetadataSelector.cs

```diff
@@ -289,10 +289,9 @@ void ValidateAuthorizationContext(EdFiAuthorizationContext authorizationContext)
                     principal.Claims.SingleOrDefault(c => c.Type == EdFiOdsApiClaimTypes.ClaimSetName)?.Value;
 
                 response.SecurityExceptionMessage =
-                    $@"Access to the resource could not be authorized. Are you missing a claim? This resource can be authorized by the following claims:
-    {string.Join(Environment.NewLine + "    ", authorizingClaimNames)}
-The API client has been assigned the '{apiClientClaimSetName}' claim set with the following resource claims:
-    {string.Join(Environment.NewLine + "    ", apiClientResourceClaims)}";
+                    $@"Access to the resource could not be authorized. The API client has been assigned the '
```

Comment:
Current formatting is
```
Access to the resource could not be authorized. The API client has been assigned the '
            ClientClaimSet' claim set. Assign different claim set which includes one of the following claims to access this resource:
            claim one
    claim two
```

fix it to be like
```
Access to the resource could not be authorized. The API client has been assigned the 'ClientClaimSet' claim set.
Assign different claim set which includes one of the following claims to access this resource:
    claim one
    claim two
```

Reply:
Even if the result is serialized to json, we are not sure how clients are parsing the message; so I think is better to return the message as readable as possible, even if that includes new line and carriage return characters once is serialized

Reply:
OK, I left them in and pushed the change.